packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -601,6 +601,14 @@ const conf = (module.exports = convict({
       format: 'nat',
     },
   },
+  recovery_key_v2: {
+    enabled: {
```

Comment:
You can leave `enabled` if you want but you could also just call it `recovery_key_v2_enabled` or like, `showRecoveryKeyV2`. I'm not sure why we have snake case here sometimes vs camelcase other times but you could go with camel in both if you wanted as well for consistency, but also OK to leave it how you've got this set up.